<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QuizBro - QuizMaster</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="box_button_control">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Button control:</string>
    </property>
    <widget class="QLabel" name="lbl_fastest_press">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Fastest press:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_press_info">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;name&gt;</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_buzzer_beater">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>211</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>&quot;Beat the buzzer&quot; Mode</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_free_buzzer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&quot;Free buzzer&quot; Mode</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_unlock_buzzers">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Unlock buzzers</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box_green_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>441</width>
      <height>73</height>
     </rect>
    </property>
    <property name="title">
     <string>Green button</string>
    </property>
    <widget class="QLabel" name="lbl_green_short_audio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Short audio:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_green_long_audio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>291</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Long audio:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_green_set_short_audio">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_green_set_long_audio">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box_yellow_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>441</width>
      <height>73</height>
     </rect>
    </property>
    <property name="title">
     <string>Yellow button</string>
    </property>
    <widget class="QLabel" name="lbl_yellow_short_audio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Short audio:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_yellow_long_audio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>291</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Long audio:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_yellow_set_short_audio">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_yellow_set_long_audio">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box_blue_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>441</width>
      <height>73</height>
     </rect>
    </property>
    <property name="title">
     <string>Blue button</string>
    </property>
    <widget class="QLabel" name="lbl_blue_short_audio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Short audio:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_blue_long_audio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>291</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Long audio:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_blue_set_short_audio">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_blue_set_long_audio">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box_red_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>441</width>
      <height>73</height>
     </rect>
    </property>
    <property name="title">
     <string>Red button</string>
    </property>
    <widget class="QLabel" name="lbl_red_short_audio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Short audio:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_red_long_audio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>291</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Long audio:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_red_set_short_audio">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_red_set_long_audio">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box_audiocontrol">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>191</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Audio control</string>
    </property>
    <widget class="QLabel" name="lbl_now_playing">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Now playing:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_audio_info">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;color/length&gt;</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_play_short">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Play short fragment</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_play_long">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Play long fragment</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="chb_mute">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Mute</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chb_simultaneous">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Allow simultaneous</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box_blinkspeed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Blink speed</string>
    </property>
    <widget class="QLabel" name="lbl_blinkspeed">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>24</y>
       <width>59</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1.0 sec</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="sld_blinkspeed">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>28</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
     <property name="singleStep">
      <number>50</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>500</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box_backgrounds">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>251</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Background - (beamer: available?)</string>
    </property>
    <widget class="QListWidget" name="lst_backgrounds">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>20</y>
       <width>250</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="scorebox_red">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>250</y>
      <width>251</width>
      <height>73</height>
     </rect>
    </property>
    <property name="title">
     <string>Red score</string>
    </property>
    <widget class="QLabel" name="lbl_red_score">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Score: 0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_red_m10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_red_m5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_red_m1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_red_p1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_red_p5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_red_p10">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+10</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="scorebox_yellow">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>330</y>
      <width>251</width>
      <height>73</height>
     </rect>
    </property>
    <property name="title">
     <string>Yellow score</string>
    </property>
    <widget class="QLabel" name="lbl_yellow_score">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Score: 0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_yellow_m10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_yellow_m5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_yellow_m1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_yellow_p1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_yellow_p5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_yellow_p10">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+10</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="scorebox_blue">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>410</y>
      <width>251</width>
      <height>73</height>
     </rect>
    </property>
    <property name="title">
     <string>Blue score</string>
    </property>
    <widget class="QLabel" name="lbl_blue_score">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Score: 0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_blue_m10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_blue_m5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_blue_m1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_blue_p1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_blue_p5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_blue_p10">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+10</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="scorebox_green">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>490</y>
      <width>251</width>
      <height>73</height>
     </rect>
    </property>
    <property name="title">
     <string>Green score</string>
    </property>
    <widget class="QLabel" name="lbl_green_score">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Score: 0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_green_m10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_green_m5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_green_m1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_green_p1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_green_p5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_green_p10">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+10</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="forbidden_answer_box">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>371</width>
      <height>553</height>
     </rect>
    </property>
    <property name="title">
     <string>Forbidden Answers</string>
    </property>
    <widget class="QComboBox" name="drd_questions">
     <property name="geometry">
      <rect>
       <x>88</x>
       <y>30</y>
       <width>271</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_question">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Question:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lst_answers">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>70</y>
       <width>370</width>
       <height>483</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
